assets/migrations/sqlite/001_initialize_schema.sql

```diff
@@ -0,0 +1,56 @@
+-- +goose Up
+CREATE TABLE tuple (
+    store CHAR(26) NOT NULL,
+    object_type VARCHAR(128) NOT NULL,
+    object_id VARCHAR(128) NOT NULL,
+    relation VARCHAR(50) NOT NULL,
+    _user VARCHAR(256) NOT NULL,
+    user_type VARCHAR(7) NOT NULL,
+    ulid CHAR(26) NOT NULL,
+    inserted_at TIMESTAMP NOT NULL,
+    PRIMARY KEY (store, object_type, object_id, relation, _user)
+);
+
+CREATE UNIQUE INDEX idx_tuple_ulid ON tuple (ulid);
+
+CREATE TABLE authorization_model (
+    store CHAR(26) NOT NULL,
+    authorization_model_id CHAR(26) NOT NULL,
+    type VARCHAR(256) NOT NULL,
+    type_definition BLOB,
+    PRIMARY KEY (store, authorization_model_id, type)
+);
```

Comment:
Interesting, I checked whether there were new migrations but didn't check to see whether existing migrations had changed since I assumed that would never be the case.

Reply:
I think I may have misinterpreted your comment earlier. I have kept this backend as close as possible to the existing implementations for MySQL and Postgres (and in fact the only change from the MySQL migrations was to separate 005 into individual ALTER TABLE statements because SQLite doesn't support altering multiple columns in a single transaction.

Now (I think) I understand that you're suggesting jumping straight to the now-current table format vs following the same set of migrations we use for MySQL and PostgreSQL, since the SQLite backend doesn't have to go through that evolution.

Reply:
Exactly! :)